lua/tests/acf/ballistics/ballistics_sv/do_round_impact.lua

```diff
@@ -9,6 +9,10 @@ return {
         State.ACF_Damage_Result = { Loss = 0, Kill = false }
         stub( ACF.Damage, "dealDamage" ).returns( State.ACF_Damage_Result )
 
+        State.DmgInfo = {}
+        State.DmgResult = {}
```

Comment:
I put these here in case we want to use them to test stuff later. As of right now, these aren't ever actually used because we stub out `dealDamage`, so we could just stub `getBulletDamage` to return nothing. Either way